docs/model_library/Hydraulics/index.rst

```diff
@@ -205,4 +214,114 @@ Allows pumping only if a pump exists in a pipeline.
         \textcolor{red}{C_{l,\tilde{l}}^{Pump}} = \textcolor{green}{\nu^{Pump}} \cdot \textcolor{red}{y_{l,\tilde{l},[t]}^{Pump}}
         + \textcolor{green}{\nu^{Electricity}} \cdot \textcolor{green}{\rho.g} \cdot \sum_{t \in T}\textcolor{red}{H_{l,\tilde{l},t}^{Pump}} \cdot \textcolor{red}{F_{l,\tilde{l},t}^{Piped}}
 
+    
+**Linearizing the Equations/constraints specific to the co_optimize method**
+
+    **Parameters**
+    :math:`\textcolor{green}{\Delta_I}` =                        Length of interval of flows for building piecewise linear model
+    :math:`M` =                       Large enough constant (maximum pressure change)
+
+    **Binary Variables**
+
+    :math:`\textcolor{red}{z_{l,\tilde{l},t, i}}, i\in \{0,1,2,..., \lceil \log_2(70000/\Delta_I) \rceil\}`:   Intermediate binary variables to determine the section of the piecewise linear graph 
```

Comment:
```suggestion
    :math:`\textcolor{red}{z_{l,\tilde{l},t, i}}, i\in \{0,1,2,..., \lceil \log_2(70000/\textcolor{green}{\Delta_I}) \rceil\}`:   Intermediate binary variables to determine the section of the piecewise linear graph 
```
I suggest always rendering \Delta_I in green to remain consistent with the convention in the documentation that parameters are always green. There are several other instances of \Delta_I throughout this section that can also be changed.

Reply:
Thanks, \Delta_I is now in green.